documentation/modules/auxiliary/gather/gitlab_tags_rss_feed_email_disclosure.md

```diff
@@ -0,0 +1,71 @@
+## Vulnerable Application
+
+Information disclosure affecting all versions of GitLab
+before 16.6.6, 16.7 prior to 16.7.4, and 16.8 prior to 16.8.1
+by sending a GET request to the project URI and appending "-/tags"
+
+## Verification Steps
+
+### Docker installation instructions can be found here:
+
+https://docs.gitlab.com/ee/install/docker.html
+
+Once installed, create a project. Once the projecte is
+created, add a new tag by expanding the Code menu item
+on the left, then selecting Tags. Then click on the 
+New Tag button in the top right corner.
```

Comment:
Move this block under Vulnerable Applications block, after the description paragraph